patches/server/0802-Configurable-feature-seeds.patch

```diff
@@ -0,0 +1,88 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Nassim Jahnke <jahnke.nassim@gmail.com>
+Date: Tue, 31 Aug 2021 17:05:27 +0200
+Subject: [PATCH] Configurable feature seeds
+
+Co-authored-by: Thonk <30448663+ExcessiveAmountsOfZombies@users.noreply.github.com>
+
+diff --git a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
+index e1f386cccdf0d2f6831c86667230e0bc8ee39cc9..da45a3ebf8554010f4eb7a00a13db6ebe76a239b 100644
+--- a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
++++ b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
+@@ -9,6 +9,7 @@ import java.util.stream.Collectors;
+ 
+ import com.google.common.collect.HashBasedTable;
+ import com.google.common.collect.Table;
++import net.minecraft.resources.ResourceLocation;
+ import net.minecraft.world.Difficulty;
+ import net.minecraft.world.entity.EntityType;
+ import net.minecraft.world.entity.monster.Vindicator;
+@@ -927,6 +928,37 @@ public class PaperWorldConfig {
+         return table;
+     }
+ 
++    public it.unimi.dsi.fastutil.objects.Object2LongOpenHashMap<ResourceLocation> featureSeeds = new it.unimi.dsi.fastutil.objects.Object2LongOpenHashMap<>();
++    private void featureSeeds() {
++        featureSeeds.defaultReturnValue(-1);
++        boolean randomise = getBoolean("feature-seeds.generate-random-seeds-for-all", false);
```

Comment:
Please use `final` on local variables.

Is it intentional for this to not be set to false again after randomising?

Reply:
We'd have to know that this is the last world to be checked to disable this, so it doesn't only generate them for one world. It doesn't have any effect afterwards (unless you go out of your way to delete entries, at which point you might as well set it to false again)